src/app/components/ATIAnalytics/types.ts

```diff
@@ -161,6 +161,21 @@ export interface ATIEventTrackingProps {
   experimentVariant?: string;
   ampExperimentName?: string;
   preventNavigation?: string;
+  componentSpecificTrackers?: ComponentSpecificTrackers;
+  blockSpecificTrackers?: BlockSpecificTrackers;
+}
+
+export interface ComponentSpecificTrackers {
```

Comment:
I am not sure if we should have the word `Specific` in the name of this interface. I wonder if we can borrow some terminology from Reverb (or WebCore)? What is this, is it an Item? Could we rename it to ItemTracker perhaps? 

```suggestion
export interface ItemTracker {
```

```suggestion
export interface ItemTracking {
```

Naming things is HARD!